tests/analysis/pdp/targets/test_shared.py

```diff
@@ -9,6 +9,26 @@ def test_df():
     return pd.DataFrame(
         {
             "student_id": ["01", "01", "01", "02", "02", "03", "04", "05"],
+            "cohort_id": [
+                "2020-21 SPRING",
+                "2020-21 SPRING",
+                "2020-21 SPRING",
+                "2021-22 FALL",
+                "2021-22 FALL",
```

Comment:
why `2020-21 SPRING` and `2021-22 FALL` repeated?

Reply:
These are just test values, and they're repeated because they're for the same student ("01" and "02", respectively). Each row is technically a "student-term", so you'll get repeated values when they're defined at the student level.

Reply:
ok, understood, great!!